src/frontend/screens/Library/components/AlphabetFilter/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useMemo, useContext } from 'react'
+import './index.css'
+import LibraryContext from '../../LibraryContext'
+
+const CHARS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ#'.split('')
+
+const AlphabetFilter: React.FC = () => {
+  const {
+    alphabetFilterLetter: currentFilter,
+    setAlphabetFilterLetter: onFilterChange,
+    gamesForAlphabetFilter: allGames
+  } = useContext(LibraryContext)
+
+  const availableChars = useMemo(() => {
+    const chars = new Set<string>()
+    if (!allGames) return chars
```

Comment:
can this ever happen? `allGames` is defined as `GameInfo[]`, so it would never be `false` nor `undefined`

are you trying to check if it's empty?

Reply:
if you mark a comment as resolved with no change at all you have to explain why

you haven't answered my questions